src/selector.rs

```diff
@@ -190,6 +190,10 @@ impl<'a, 'o, 'i> parcel_selectors::parser::Parser<'i> for SelectorParser<'a, 'o,
       "corner-present" => WebKitScrollbar(WebKitScrollbarPseudoClass::CornerPresent),
       "window-inactive" => WebKitScrollbar(WebKitScrollbarPseudoClass::WindowInactive),
 
+      "local" | "global" => {
```

Comment:
Should we only error if CSS modules are enabled and just pass it through otherwise? Not sure if other tooling might be relying on passthrough of unknown selectors for some reason.

```suggestion
      "local" | "global" if self.options.css_modules.is_some() => {
```